installer/scripts/omsadmin.sh

```diff
@@ -787,6 +804,36 @@ show_workspace_status()
     fi
 }
 
+show_omsagent_proc_limit()
+{
+    llimit=-1
+    resultcode=0
+
+    count=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | wc -l`
+
+    if [ "$count" -eq "0" ]; then
+        log_info "INFO:  OMS Agent process limit NOT Presently Set."
+    elif [ $count -eq 1 ]; then
+        llimit=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | awk '{print $4}'`
+    else
+        log_warning "$count $AGENT_USER entries found in $PROC_LIMIT_CONF.  This may yield problems.  Last entry should apply."
+        log_warning ">>Last entry presumed to apply<<"
+        grep -E "$LIMIT_LINE_REGEX" $PROC_LIMIT_CONF 1>&2
+        log_warning "End of the $count duplicate entry WARNING Message"
+        llimit=`grep -E "$LIMIT_LINE_REGEX" $PROC_LIMIT_CONF | tail -1 | awk '{print $4}'`
+    fi
+
+    if [ "$llimit" -eq -1 ]; then
+        log_info "$AGENT_USER proc count unlimited."
```

Comment:
Return in this case; you should not print out a limit of -1

Reply:
Ok.

Reply:
So in that case, I will also make the initial case result_limit='', as that will fully demonstrate the intention of the routine, and then the single spot of the conditional on -1 will indicate the special case.